tools/browserutils/browser.py

```diff
@@ -0,0 +1,268 @@
+
+        dest = os.path.join(dest, "profiles")
+        if not os.path.exists(dest):
+            os.makedirs(dest)
+        with open(os.path.join(dest, "prefs_general.js"), "wb") as f:
+            resp = get("https://hg.mozilla.org/mozilla-central/raw-file/tip/testing/profiles/prefs_general.js")
+            f.write(resp.content)
+
+        return dest
+
+    def _latest_geckodriver_version(self):
+        """Get and return latest version number for geckodriver."""
+        # This is used rather than an API call to avoid rate limits
+        tags = call("git", "ls-remote", "--tags", "--refs",
+                    "https://github.com/mozilla/geckodriver.git")
+        release_re = re.compile(".*refs/tags/v(\d+)\.(\d+)\.(\d+)")
+        latest_release = 0
+        for item in tags.split("\n"):
+            m = release_re.match(item)
+            if m:
+                version = [int(item) for item in m.groups()]
+                if version > latest_release:
+                    latest_release = version
+        assert latest_release != 0
+        return "v%s.%s.%s" % tuple(str(item) for item in latest_release)
+
+    def install_webdriver(self, dest=None):
+        """Install latest Geckodriver."""
+        if dest is None:
+            dest = os.pwd
+
+        version = self._latest_geckodriver_version()
+        logger.debug("Latest geckodriver release %s" % version)
+        url = ("https://github.com/mozilla/geckodriver/releases/download/%s/geckodriver-%s-%s.tar.gz" %
+               (version, version, self.platform_string_geckodriver()))
+        untar(get(url).raw, dest=dest)
+        return find_executable(os.path.join(dest, "geckodriver"))
+
+    def version(self, root):
+        """Retrieve the release version of the installed browser."""
+        platform_info = RawConfigParser()
+
+        with open(os.path.join(root, self.platform_ini), "r") as fp:
+            platform_info.readfp(BytesIO(fp.read()))
+            return "BuildID %s; SourceStamp %s" % (
+                platform_info.get("Build", "BuildID"),
+                platform_info.get("Build", "SourceStamp"))
+
+
+class Chrome(Browser):
+    """Chrome-specific interface.
+
+    Includes installation, webdriver installation, and wptrunner setup methods.
+    """
+
+    product = "chrome"
+    binary = "/usr/bin/google-chrome"
+    requirements = "requirements_chrome.txt"
+
+    def platform_string(self):
+        platform = {
+            "Linux": "linux",
+            "Windows": "win",
+            "Darwin": "mac"
+        }.get(uname[0])
+
+        if platform is None:
+            raise ValueError("Unable to construct a valid Chrome package name for current platform")
+
+        if platform == "linux":
+            bits = "64" if uname[-1] == "x86_64" else "32"
+        elif platform == "mac":
+            bits = "64"
+        elif platform == "win":
+            bits = "32"
+
+        return "%s%s" % (platform, bits)
+
+    def install(self):
+        return None
```

Comment:
I know we don't invoke this method today, but it seems like for the time being, we'd want to inherit the base class's implementation.